docker-compose.override.yml.sample

```diff
@@ -4,6 +4,19 @@ services:
     environment:
       HOST_IP: 172.22.26.1
 
+  # in ports override IP address of docker0 based on your local machine IP.
+   dns:
```

Comment:
There's no need to include common specs in override file. Just specify the different parts (which here is **ports**). I mean don't specify restart, image, command, volumes, cap_add specs in override file